Dockerfile

```diff
@@ -10,7 +10,7 @@ WORKDIR /app
 COPY ${JAR_LIB_FILE} app.jar
 
 # Run the api
-CMD ["java", "-jar", "app.jar"]
+CMD ["java", "-agentlib:jdwp=transport=dt_socket,address=*:6006,server=y,suspend=n", "-jar", "app.jar"]
```

Comment:
Is this safe to run in production?  This `Dockerfile` is used when running our application in production.  Does it also not degrade performance?

Reply:
Yes, it can degrade performance. I'll exclude it.